### PR TITLE
Add e2e test for labels refund

### LINKED ISSUE #2011: Test refunding a label
**Test the following use case:**
User has previously purchased a label but would like to refund.
image

**Test should include:**
- Verify "request refund" works if label has not been used.
- Verify "request refund" fails if label is already used. 
- Verify the user can purchase a label again after a successful refund.

### PR DESCRIPTION
**Pull request by Ferdev**
Fixes #2011

### CODE REVIEW

**Comment by superdav42** [comment]
:eyes:

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "globby": "^6.1.0",
     "jest": "^24.9.0",
     "jest-puppeteer": "^4.4.0",
+    "moment": "^2.25.3",
```
Comment (superdav42) [inline package.json:49]: I think this should be a dev dependency since it's just used be the e2e test.

Comment (Ferdev) [reply package.json:49]: yeah! It's in `devDependencies` :smiley:

**Comment by c-shultz** [comment]
Is this ready to merge, @Ferdev ?

**Comment by Ferdev** [comment]
Yeah it just seems the review is pending.

**Comment by c-shultz** [comment]
@superdav42, did you have any follow up comments or is this good to go on your end?

**Pull request merged**